src/formats/qoi.c

```diff
@@ -0,0 +1,135 @@
+// SPDX-License-Identifier: CC0
```

Comment:
It is a custom implementation of QOI decodes, so lets use the project's license MIT.